backend/src/modules/sendEmails/sendEmailsModuleCore.ts

```diff
@@ -0,0 +1,17 @@
+import { ModuleCore } from '../../shared/Module/core/ModuleCore';
+import { SendEmail } from './application/SendEmailCommand';
+import { SendEmailCommandHandler } from './application/SenEmailCommandHandler';
+import { EmailSender } from './application/EmailSender';
+
+export function SendEmailModuleCore(emailSender: EmailSender): ModuleCore {
```

Comment:
Tutaj nasz "Core" będzie otrzymywał taki zestaw propsów (jak na screenie), z których będzie mógł skorzystać. Do czego mogą one posłużyć?

- eventPublisher - czyli takie coś co pozwala nam dodawać eventy, oraz nasłuchiwać na nie
- commandPublisher, - to samo co z eventami ale tyczy się komend
-  currentTimeProvider, - otrzymasz czas jeśli wywołasz metodę z tego propsa
-  entityIdGenerator, - a tu podonie jak z czasem, tylko otrzymasz indywidualny ID
-  repository, - tu pożesz(ale nie musisz u siebie) zamieścić repozytorium, które przekażesz dalej
  
Słowo   "publisher" możemy wymiennie stosować tutaj ze słowem "bus" (dla uproszczenia).

  



![image](https://user-images.githubusercontent.com/41823689/168163553-e0fef3bd-59a5-4563-8027-37b6d806501d.png)